src/chapters/exceptions/exercises.rst

```diff
@@ -0,0 +1,29 @@
+Exercises
```

Comment:
This should be Exercises: <section topic>. check the other exercise headers for the format

Reply:
Fixed!